Containerfile.cli

```diff
@@ -0,0 +1,36 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.4 AS builder
```

Comment:
This should be changed to brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.6

Reply:
Thank you